src/main/java/edu/itu/cavabunga/exception/ParticipantNotFound.java

```diff
@@ -0,0 +1,17 @@
+package edu.itu.cavabunga.exception;
+
+public class ParticipantNotFound extends RuntimeException {
```

Comment:
We need a base not found class which extends runtimeexception, and maybe this classes extends base notfoundclass. Also, still I am not sure to do we need many exception class for different objects.Maybe just one notfoundclass is enough, since we can specify which component not found in message.

Reply:
Fixed in c76d5c951340f9257667185e15f4522b01b74593